src/app/api/v1/user/email/route.ts

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getToken } from "next-auth/jwt";
+import { NextRequest, NextResponse } from "next/server";
+import AppConstants from "../../../../../appConstants";
+
+import {
+  getSubscriberByEmail,
+  deleteResolutionsWithEmail,
+} from "../../../../../db/tables/subscribers";
+import {
+  addSubscriberUnverifiedEmailHash,
+  removeOneSecondaryEmail,
+  getEmailById,
+} from "../../../../../db/tables/emailAddresses.js";
+
+import { sendVerificationEmail } from "../../../utils/email";
+
+import { validateEmailAddress } from "../../../../../utils/emailAddress";
+import { getL10n } from "../../../../functions/server/l10n";
+import { initEmail } from "../../../../../utils/email";
+
+interface EmailAddRequest {
+  email: string;
+}
+
+export async function POST(req: NextRequest) {
+  const token = await getToken({ req });
+  const l10n = getL10n();
+
+  if (token) {
+    try {
+      const body: EmailAddRequest = await req.json();
+      const subscriber = await getSubscriberByEmail(token.email);
+      const emailCount = 1 + (subscriber.email_addresses?.length ?? 0); // primary + verified + unverified emails
+      const validatedEmail = validateEmailAddress(body.email);
+
+      if (validatedEmail === null) {
+        return NextResponse.json({
+          success: false,
+          status: 400,
```

Comment:
should the `status` be here or in the second arg?
https://nextjs.org/docs/app/api-reference/functions/next-response#json

Same w/ L51, L61, L70 below:

https://github.com/mozilla/blurts-server/blob/82c28e2f27eca0cfcafad455107be185dd0dd8cc/src/app/api/v1/user/email/route.ts#L91-L97

Reply:
Likewise: 52cb724cf